autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -888,12 +889,29 @@ def get_feature_importance(self, model=None, X=None, y=None, features=None, raw=
     #  This is different from raw, where the predictions of the folds are averaged and then feature importance is computed.
     #  Consider aligning these methods so they produce the same result.
     # The output of this function is identical to non-raw when model is level 0 and non-bagged
-    def _get_feature_importance_raw(self, model, X, y, features_to_use=None):
+    def _get_feature_importance_raw(self, model, X, y, features_to_use=None, silent=False):
+        time_start = time.time()
         model: AbstractModel = self.load_model(model)
         if features_to_use is None:
             features_to_use = list(X.columns)
+        feature_count = len(features_to_use)
+
+        if not silent:
+            logger.log(20, f'Computing raw permutation importance for {feature_count} features on {model.name} ...')
+
+        sample_size = 10000
```

Comment:
Should this always do subsampling, or only when user did not provide a dataset?

Also should this be user-facing arg with 10000 as default value?

Reply:
Added